lldb/test/API/tools/lldb-server/TestGdbRemoteLaunch.py

```diff
@@ -78,6 +93,7 @@ def test_launch_via_vRun_no_args(self):
         self.expect_gdbremote_sequence()
 
     @add_test_categories(["llgs"])
+    @skipIfRemote
```

Comment:
Why is only this test skipped?

Reply:
Ah, because we need to open the file while we attempt to run it, and doing that on the remote is difficult.

We could run some shell command, but it'd be a different one per remote platform, and failing that, Python. Assuming the remote even has an OS.

So yeah, skipIfRemote is logical here :)